cmd/state-custom-installer/docker/centos-7/runCompile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd cli
+git pull
+git checkout BE-1031-there-is-a-standalone-tool-for-unpacking-and-relocating-artifacts-in-installers 
```

Comment:
If this is an alternate branch, there should be a tag instead.

Reply:
No, waiting for assimilation to Master.

Reply:
Changed the script to assume we are running from Master.